flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-kafka/src/main/java/org/apache/flink/cdc/connectors/kafka/json/canal/CanalJsonSerializationSchema.java

```diff
@@ -200,14 +211,19 @@ public byte[] serialize(Event event) {
         }
     }
 
+    /**
+     * Refer to <a
+     * href="https://github.com/alibaba/canal/blob/9373429015c0f25318b703833a1d7913676f2aa3/protocol/src/main/java/com/alibaba/otter/canal/protocol/FlatMessage.java#L74">...</a>.
+     */
```

Comment:
Since Canal has no official protocol docs, will it be better if we change the link to [Canal | Apache Flink](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/formats/canal/#available-metadata)?